packages/esm-patient-banner-app/src/index.ts

```diff
@@ -28,6 +28,8 @@ export const deceasedPatientTag = getSyncLifecycle(deceasedPatientTagComponent,
 
 export const patientBanner = getSyncLifecycle(patientBannerComponent, options);
 
+export const printIdentifierSticker = getSyncLifecycle(printIdentifierStickerComponent, options);
```

Comment:
```suggestion
export const printIdentifierSticker = getSyncLifecycle(printIdentifierStickerComponent, options);
```

`getSyncLifecycle` or `getAsyncLifecycle` here, @ibacher? I forget.

Reply:
@denniskigen, I always prefer alining components like the already existing to avoid hours of debugging.  Since all elements in `esm-banner` are using `getSyncLifecycle` i decided to use that too

Reply:
Thanks, @ibacher.